app/component/search/search-modal.cjsx

```diff
@@ -6,70 +6,38 @@ FormattedMessage = intl.FormattedMessage
 SearchInput      = require './search-input'
 SearchActions    = require '../../action/search-actions'
 EndpointActions  = require '../../action/endpoint-actions'
+Tabs             = require 'material-ui/lib/tabs/tabs'
+Tab              = require 'material-ui/lib/tabs/tab'
+config           = require '../../config'
 
 class SearchModal extends React.Component
 
   @contextTypes:
     getStore: React.PropTypes.func.isRequired
     executeAction: React.PropTypes.func.isRequired
 
-  componentWillMount: =>
-    @context.getStore('SearchStore').addChangeListener @onSearchChange
-
-  componentWillUnmount: =>
-    @context.getStore('SearchStore').removeChangeListener @onSearchChange
-
-  onSearchChange: (a) =>
-    position = @context.getStore('SearchStore').getPosition()
-    @setState("value": position?.address || "")
-
-  closeModal: () =>
-    @context.executeAction SearchActions.closeSearch
-
   render: =>
-    style = {}
-    if @context.getStore('SearchStore').isModalOpen() == false
-      style.left = "-400%"
-    else
-      style.right = "0px"
-
-    <div style={style}
-      className="search-modal">
-      <div className="row">
-        <div className="small-12 medium-6 medium-offset-3 columns cursor-pointer search-header" onClick={@closeModal}>
-          <Icon img={'icon-icon_arrow-left'} />
-          <span className="search-header-separator"/>
+    if @props.modalIsOpen
```

Comment:
Just a nitpick, but it is usually nice to exit immediately rather than in else, i.e. transforming this to

```
if !@props.modalIsOpen
  return false
<div ...
```